### PR TITLE
Display exact mon_start error reason in GUI popup 

### PR DESCRIPTION
**Pull request by gomoripeti**

### CODE REVIEW

File: apps/xprof_gui/priv/src/actions/NotificationActions.js [hunk 1]
```diff
@@ -43,8 +44,10 @@ export const addNotification = (
   const lastId = getNotificationsLastId(state);
   const notification = {
     id: lastId + 1,
-    severity,
-    message,
+    severity: (error && error.data && error.data.severity)
```
Comment (gomoripeti) [inline apps/xprof_gui/priv/src/actions/NotificationActions.js]: is there a nicer way to check whether a deep object field is defined?

**Comment by gomoripeti** [comment]
TODO: handle 204 "the requested MFA is already monitored" also (as this is a 2xx code, this might not be handled as an error by ApiUtils)

File: apps/xprof_gui/priv/src/actions/NotificationActions.js (hunk 1 above)
Comment (MarcinKasprowicz) [reply apps/xprof_gui/priv/src/actions/NotificationActions.js]: you can use lodash -` _.get(error, 'error.data.severity'))`, however it isn't nicer :P

Comment (gomoripeti) [reply apps/xprof_gui/priv/src/actions/NotificationActions.js]: 😄  actually with a default value its more compact and less redundant, I like it
```
_.get(error, 'data.severity', defaultSeverity)
```

**Pull request merged**